libtransmission/rpc-server.c

```diff
@@ -560,12 +560,31 @@ static void handle_request(struct evhttp_request* req, void* arg)
 
     if (req != NULL && req->evcon != NULL)
     {
+        static int attempts = 0;
```

Comment:
If someone was to create two RPC servers, they would interfere with each other. There is a `tr_rpc_server` structure at the top of this file that holds the server state and this field should be moved there (and probaly renamed to give an idea of what we're counting exactly). The structure is available in the scope of this function as `server`.